spec/src/modules/quizzes.js

```diff
@@ -92,21 +92,42 @@ describe(`ConstructorIO - Quizzes${bundledDescriptionSuffix}`, () => {
       });
     });
 
-    it('Should return a result provided a valid apiKey, quizId and versionId', () => {
-      const versionId = 'e03210db-0cc6-459c-8f17-bf014c4f554d';
+    it('Should return a result provided a valid apiKey, quizId and quizVersionId', () => {
+      const quizVersionId = 'e03210db-0cc6-459c-8f17-bf014c4f554d';
       const { quizzes } = new ConstructorIO({
         apiKey: quizApiKey,
         fetch: fetchSpy,
       });
 
-      return quizzes.getQuizNextQuestion(validQuizId, { versionId }).then((res) => {
+      return quizzes.getQuizNextQuestion(validQuizId, { quizVersionId }).then((res) => {
         const requestedUrlParams = helpers.extractUrlParamsFromFetch(fetchSpy);
 
-        expect(res).to.have.property('version_id').to.be.an('string').to.equal(versionId);
+        expect(res).to.have.property('version_id').to.be.an('string').to.equal(quizVersionId);
```

Comment:
I think we'll want to change the expectation to use `quiz_version_id` as well right?